packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

```diff
@@ -59,6 +57,24 @@ Tests can be executed in any of the following supported form factors:
 | huge     | 1300  | 768    |
 | enormous | 1500  | 768    |
 
+## Environment Variables
+
+The following environment variables are configurable for the test run. Variables marked as reserved will be overwritten by the test runner. To configure reserved variables use the test runner configuration options. If the test runner is not being used the variables are configurable by the user.
+
+| Variable Name                 | Type    | Reserved | Default             | Description                                                                                                                               |
+|-------------------------------|---------|----------|---------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| BROWSERS                      | string  | true     | chrome              | A list of browsers for the test run. Browsers are delimited by a comma. Example: chrome,firefox,ie                                        |
+| FORM_FACTOR                   | string  | true     |                     | A form factor assigned to the test run.                                                                                                   |
```

Comment:
Do we also want to call out that the default is `huge`? https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/services/wdio-terra-service.js#L99

I don't know, what do you think?

Reply:
Hmm, this is a tricky one. It's a bit of a gray area. When a FORM_FACTOR is set only that form factor is allowed to run. When no FORM_FACTOR is set we loop through all form factors using the describeViewports utility.

Reply:
I went ahead and specified this as `huge`:

https://github.com/cerner/terra-toolkit/pull/544/commits/eb863d146f8d4cdc9ac5c7e661bd2830bf548a8f

Reply:
Or should we leave it blank and just update the description to include what you just said and that only if that form_factor is one of the form factors listed in describeViewports?

Reply:
Ok. that's fine. I didn't see your latest comment before adding my latest comment.